paasta_tools/cli/cmds/local_run.py

```diff
@@ -826,7 +827,11 @@ def run_docker_container(
     if interactive or not simulate_healthcheck:
         # NOTE: This immediately replaces us with the docker run cmd. Docker
         # run knows how to clean up the running container in this situation.
-        execlpe("paasta_docker_wrapper", *docker_run_cmd, merged_env)
+        wrapper_path = shutil.which("paasta_docker_wrapper")
+        # To properly simulate mesos, we pop the PATH, which is not available to
+        # The executor
+        merged_env.pop("PATH")
```

Comment:
Wouldn't the more correct thing be to just get rid of `merged_env` entirely and only use `environment`?

Reply:
I thought about that, but then remembers that we need some `environment` variables because that is how we do secrets. 

If you want I can make a function that strips out everything from the environment *except* PAASTA_ vars, and then use that instead? (instead of explicility popping PATH)

Reply:
Doesn't paasta local-run set the PAASTA_ variables?

Reply:
I think I better understand what you were getting at here. Let me retry.